bridge/engine/EngineStreamDirector.h

```diff
@@ -693,20 +772,6 @@ class EngineStreamDirector
         return oldHighestActiveIndex != simulcastStream._highestActiveLevel;
     }
 
```

Comment:
I remember something about quick way to figure out when to forward dominant speaker as hi-res. 
Are we tracking the complexity of the director decisions?